ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -290,7 +314,9 @@ export const useMhrInformation = () => {
       addOwnerGroups: isTransferDueToDeath.value
         ? await parseDueToDeathOwnerGroups(isDraft)
         : await parseOwnerGroups(isDraft),
-      deleteOwnerGroups: await parseDeletedOwnerGroups()
+      deleteOwnerGroups: isTransferToExecutorProbateWill.value
+        ? await parseDeletedWillOwnerGroups()
+        : await parseDeletedOwnerGroups()
```

Comment:
Good idea to create new methods.
Depending on the variations between the last few transfer type payloads, we could create a function to switch between each according to transfer type. 
This works for now!

Reply:
Will rename for easier changes in future :)